conda_forge_tick/cli.xsh

```diff
@@ -0,0 +1,35 @@
+import argparse
+import time
+
+from doctr.travis import run_command_hiding_token as doctr_run
+
+from .all_feedstocks import main as main0
+from .make_graph import main as main1
+from .update_upstream_versions import main as main2
+from .auto_tick import main as main3
+
+int_script_dict = {0: main0, 1: main1, 2: main2, 3: main3}
```

Comment:
Also known as a list :) `[main0, main1, ...]`

Reply:
We might come up with more descriptive names at some point, you know if we got around to it.

Reply:
:+1: